src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -51,7 +51,13 @@ public Task Stats()
 
             return Task.CompletedTask;
         }
-
+        
+        [RavenAction("/databases/*/is-loaded", "GET", AuthorizationStatus.ValidUser, IsDebugInformationEndpoint = true)]
+        public Task IsLoaded()
+        {
+            return Task.CompletedTask;
```

Comment:
Please return something here. This is a debug endpoint, and we expect some JSON

Reply:
I think it would be better to rename it to `/databases/*/alive` or `/databases/*/healthcheck` and return no-content. There is no need to return anything.